applications_tests/lethe-particles/depreciated_parameters.prm

```diff
@@ -0,0 +1,101 @@
+# Listing of Parameters
```

Comment:
Same here, right?. I believe the name of the file should be "deprecated" instead of "depreciated"